content/en/docs/configuring-sdk/custom-http.md

```diff
@@ -40,7 +82,52 @@ See <https://golang.org/pkg/net/#Dialer.Timeout>
 
 We'll call this ``Connect`` as **time.Duration**.
 
-## Transport.ExpectContinueTimeout
+## Transport
+
+The `BuildableHTTPClient` provides a builder mechanics for constructing clients with modified 
+[Transport](https://golang.org/pkg/net/http#Transport) options.
+
+### Configuring a Proxy
+
+If you cannot directly connect to the internet, you can use Go-supported
+environment variables (`HTTP_PROXY` / `HTTPS_PROXY`) or create a custom HTTP client to
+configure your proxy. The following example will configure the client to use `PROXY_URL` as the proxy
+endpoint:
+
+```go
+import awshttp "github.com/aws/aws-sdk-go-v2/aws/transport/http"
+import "net/http"
+
+// ...
+
+httpClient := awshttp.NewBuildableClient().WithTransportOptions(func(tr *http.Transport) {
+	proxyURL, err := url.Parse("PROXY_URL")
+	if err != nil {
+		log.Fatal(err)
+	}
+	tr.Proxy = http.ProxyURL(proxyURL)
+})
+```
+
+### Other Settings
+
+Below are a few other `Transport` settings that can be modified to tune the HTTP client. These settings can be applied
+as shown in the example below: Additional settings not described here can be found in the
```

Comment:
"example below" -> "following example"

"Additional settings" -> "Any additional settings that are"